pkg/dao/suite_test.go

```diff
@@ -45,27 +45,34 @@ type RepositorySuite struct {
 
 var orgIDTest = seeds.RandomOrgId()
 var accountIdTest = seeds.RandomAccountId()
+var timestamp = time.Now()
 
 var repoPublicTest = models.Repository{
 	Base: models.Base{
 		CreatedAt: time.Now(),
 		UpdatedAt: time.Now(),
 	},
-	URL:           "https://www.redhat.com",
-	LastReadTime:  nil,
-	LastReadError: nil,
-	Public:        true,
+	URL:                          "https://www.redhat.com",
+	Public:                       true,
+	LastIntrospectionTime:        &timestamp,
+	LastIntrospectionUpdateTime:  &timestamp,
+	LastIntrospectionSuccessTime: &timestamp,
+	LastIntrospectionError:       nil,
+	Status:                       StatusValid,
 }
 
 var repoPrivateTest = models.Repository{
 	Base: models.Base{
 		CreatedAt: time.Now(),
 		UpdatedAt: time.Now(),
 	},
-	URL:           "https://www.redhat.private.com",
-	LastReadTime:  nil,
-	LastReadError: nil,
-	Public:        false,
+	URL:                          "https://www.redhat.private.com",
```

Comment:
This URL is not safe!
You could use, private.redhat.com, but try to use "example.com.", "example.net.", and "example.org." with subdomains . See: https://www.rfc-editor.org/rfc/rfc6761.html